code/web/src/modules/user/EditProfile.js

```diff
@@ -94,10 +107,19 @@ class EditProfile extends Component {
 
             <form>
               <input type="file" onChange={this.onUpload}></input>
-
               <input value={this.props.user.details.email}></input>
-
-                <Button onClick={this.onSubmit} theme="primary" style={{ marginLeft: '1em' }}>Save</Button>
+              <label for="description">Enter your User Description:</label>
+              {/* <textarea onChange={this.onChange} name="description" id="description" value={this.props.user.details.description} rows="4" cols="50"></textarea> */}
```

Comment:
Since we know the Textarea stylized component is the route to go, we'll want to remove this line.